packages/commonwealth/client/scripts/views/composition/MobileNavigation/NavigationButton/utils.ts

```diff
@@ -0,0 +1,18 @@
+import { IconName } from 'views/components/component_kit/cw_icons/cw_icon_lookup';
+
+import { NavigationButtonProps } from '../NavigationButton';
+
+export const typeToIconAndName = (
```

Comment:
i think we should add some comments here

Reply:
added